api/lib/source/polkadotV0-source.js

```diff
@@ -711,16 +711,19 @@ class polkadotAPI {
     }
   }
 
-  getDemocracyProposalDetailedVotes(proposal, links) {
+  async getDemocracyProposalDetailedVotes(proposal, links) {
+    const api = await this.getAPI()
+
     // in democracy proposals there is the first opening deposit made by the proposer
     // afterwards every account that seconds the proposal must deposit the same amount from the initial deposit
     const depositsSum = toViewDenom(
       this.network,
       BigNumber(proposal.balance).times(proposal.seconds.length).toNumber()
     )
+    const depositerInfo = await api.derive.accounts.info(proposal.proposer)
     const deposits = [
       {
-        depositer: proposal.proposer,
+        depositer: this.reducers.networkAccountReducer(depositerInfo),
```

Comment:
instead of this being a unique key, why not just `identity` or `networkAccount` for all?